service/src/main/java/bio/terra/cbas/dependencies/dockstore/DockstoreClient.java

```diff
@@ -0,0 +1,24 @@
+package bio.terra.cbas.dependencies.dockstore;
+
+import bio.terra.cbas.config.DockstoreServerConfiguration;
+import bio.terra.dockstore.client.ApiClient;
+import javax.ws.rs.client.Client;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DockstoreClient {
+
+  private final DockstoreServerConfiguration dockstoreServerConfiguration;
+  private final Client commonHttpClient = new ApiClient().getHttpClient();
```

Comment:
Follow same convention of using singleton client. See [this PR](https://github.com/DataBiosphere/cbas/pull/164/files#diff-0e12086dab69c580ca388826597f9e392e5f42410050a69151ed5a9002164831).

Reply:
Updated the code